homeassistant/components/jenkins/sensor.py

```diff
@@ -0,0 +1,82 @@
+"""This component provides a HA sensor for Jenkins."""
+
+from jenkinsapi.jenkins import Jenkins
+from requests.exceptions import HTTPError
+
+from homeassistant.const import CONF_HOST, CONF_TOKEN, CONF_USERNAME
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.entity import Entity
+
+from . import _LOGGER
+from .const import CONF_JOB_NAME, DOMAIN
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up Jenkins sensor based on a config entry."""
+    try:
+        jenkins = await hass.async_add_executor_job(
+            Jenkins,
+            entry.data[CONF_HOST],
+            entry.data[CONF_USERNAME],
+            entry.data[CONF_TOKEN],
+        )
+        _LOGGER.debug("Successfully connected to %s", entry.data[CONF_HOST])
+    except HTTPError as exception:
+        raise PlatformNotReady from exception
+
+    [repository, branch] = entry.data[CONF_JOB_NAME].split("/", maxsplit=1)
+    job = await hass.async_add_executor_job(jenkins.get_job, entry.data[CONF_JOB_NAME])
+    sensor = JenkinsSensor(repository, branch, job)
+
+    async_add_entities([sensor], True)
```

Comment:
I think you can remove the `True` argument here, since you don't need to update before adding to the entity registry.